src/App.js

```diff
@@ -48,6 +53,7 @@ class App extends React.Component {
           onChange={this.handleInput} 
         />
       </div>
+      </>
```

Comment:
What's this?

Reply:
Earlier everything was inside a <div> and CSS was applied on the whole div. I created a div outside that and since we can return only one thing, I wrapped everything inside a empty element <> . . . </>

Reply:
Oh! I didn't know empty element `<></>` was a thing :P